plugins/modules/vmware_datastore_cluster.py

```diff
@@ -272,14 +641,34 @@ def main():
             keep_vmdks_together=dict(type='bool', default=True, required=False),
             automation_level=dict(type='str', choices=['automated', 'manual'], default='manual'),
             enable_io_loadbalance=dict(type='bool', default=False, required=False),
-            loadbalance_interval=dict(type='int', default=480, required=False)
+            loadbalance_interval=dict(type='int', default=480, required=False),
+            # Automation overrides
+            space_balance_automation_level=dict(type='str', choices=['automated', 'manual', 'cluster_settings']),
+            rule_enforcement_automation_level=dict(type='str', choices=['automated', 'manual', 'cluster_settings']),
+            policy_enforcement_automation_level=dict(type='str', choices=['automated', 'manual', 'cluster_settings']),
+            vm_evacuation_automation_level=dict(type='str', choices=['automated', 'manual', 'cluster_settings']),
+            # Space Load Balance Config
+            min_space_utilization_difference=dict(type='int', required=False),
+            free_space_threshold_gb=dict(type='int', required=False),
+            space_utilization_threshold=dict(type='int', required=False),
+            # IO Load Balance Config
+            io_balance_automation_level=dict(type='str', choices=['automated', 'manual', 'cluster_settings']),
+            io_latency_threshold=dict(type='int', required=False),
+            io_load_imbalanc_threshold=dict(type='int', required=False),
+            # VM overrides
+            vm_overrides=dict(type='list', elements='dict', required=False, options=dict(
+                vm_name=dict(type='str', required=True),
+                keep_vmdks_together=dict(type='bool', default=None),
+                automation_level=dict(type='str', choices=['none', 'automated', 'manual', 'disabled'], default='none'))
+                )
```

Comment:
```suggestion
            )
```

ERROR: Found 1 pep8 issue(s) which need to be resolved:
ERROR: plugins/modules/vmware_datastore_cluster.py:663:17: E123: closing bracket does not match indentation of opening bracket's line (100%)